python/ray/train/base_trainer.py

```diff
@@ -82,9 +82,8 @@ class BaseTrainer(abc.ABC):
       called in sequence on the remote actor.
     - ``trainer.setup()``: Any heavyweight Trainer setup should be
       specified here.
-    - ``trainer.preprocess_datasets()``: The provided
-      ray.data.Dataset are preprocessed with the provided
-      ray.data.Preprocessor.
+    - ``trainer.preprocess_datasets()``: The datasets passed to the Trainer will be
+      setup and assigned to workers.
```

Comment:
"assigned to workers" only applies to `DataParallelTrainer`, not `BaseTrainer`. Non data-parallel trainers like `SklearnTrainer` don't have this splitting logic.